src/functionalities/Functionality.js

```diff
@@ -36,6 +37,10 @@ class Functionality {
     })
   }
 
+  static get _NON_TEMPLATE_KEYS() {
+    return [ 'id', 'name' ]
+  }
```

Comment:
as an example to what I mean in the PR body, my desire was to make this easily extended in children like `HeidolphMagneticStirrer`